src/PrestaShopBundle/Resources/views/Admin/macros.html.twig

```diff
@@ -89,19 +89,48 @@
 {% endmacro %}
 
 {# Show form widget with errors rendered below it #}
-{% macro form_widget_with_error(form, vars) %}
+{% macro form_widget_with_error(form, vars, extraVars) %}
   {% set vars = vars|default({}) %}
-  {% set attr = vars.attr|default({}) %}
-  {% set errors = form.vars.errors %}
-
-  {% set attr = attr|merge({'class': (attr.class is defined ? attr.class : '') ~ (errors|length > 0 ? ' is-invalid' : '')} ) %}
-  {% set vars = vars|merge({'attr': attr}) %}
+  {% set extraVars = extraVars|default({}) %}
 
   {{ form_widget(form, vars) }}
 
+  {% if extraVars.help is defined %}
+    <small class="form-text">{{ extraVars.help }}</small>
+  {% endif %}
+
   {% for error in form.vars.errors %}
-    <div class="invalid-feedback">
+    <div class="text-danger">
       {{ error.message }}
     </div>
   {% endfor %}
 {% endmacro %}
+
+ {#
+  Helper function to render most common structure for single input
+  @param form - form view to render
+  @param vars - custom vars that are passed to form_widget
+  @param form - parameters that are not related to form_widget, but are needed for input (label, help text & etc.)
+ #}
+{% macro form_group_row(form, vars, extraVars) %}
```

Comment:
Isn't it a duplicate of https://github.com/PrestaShop/PrestaShop/blob/develop/src/PrestaShopBundle/Resources/views/Admin/macros.html.twig#L116 ?

Reply:
it is, since it wasnt merged at that time, i just did copy-paste to have it in this PR as well.